### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by erciccione**
@serhack do you mind giving it a review?

### CODE REVIEW

File: translations/monero-core_it.ts [hunk 1]
```diff
@@ -310,123 +310,123 @@
...
         <location filename="../pages/Keys.qml" line="193"/>
         <source>Secret spend key</source>
-        <translation type="unfinished">Chiave segreta di spesa </translation>
+        <translation>Chiave segreta spendibile </translation>
     </message>
     <message>
         <location filename="../pages/Keys.qml" line="194"/>
         <source>Public spend key</source>
-        <translation type="unfinished">Chiave di spesa pubblica</translation>
+        <translation>Chiave spendibile pubblica</translation>
```
Comment (serhack) [inline translations/monero-core_it.ts]: Chiave pubblica spendibile

File: translations/monero-core_it.ts [hunk 2]
```diff
@@ -310,123 +310,123 @@
...
         <location filename="../pages/Keys.qml" line="192"/>
         <source>Public view key</source>
-        <translation type="unfinished">Chiave di visualizzazione pubblica</translation>
+        <translation>Chiave di visualizzazione pubblica</translation>
     </message>
     <message>
         <location filename="../pages/Keys.qml" line="193"/>
         <source>Secret spend key</source>
-        <translation type="unfinished">Chiave segreta di spesa </translation>
+        <translation>Chiave segreta spendibile </translation>
```
Comment (serhack) [inline translations/monero-core_it.ts]: Why not "Privata" ?

File: translations/monero-core_it.ts [hunk 3]
```diff
@@ -519,12 +519,12 @@
...
         <location filename="../LeftPanel.qml" line="508"/>
         <source>Seed &amp; Keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Chiavi &amp; Seed</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="509"/>
         <source>Y</source>
-        <translation type="unfinished"></translation>
+        <translation>Y</translation>
```
Comment (serhack) [inline translations/monero-core_it.ts]: ?

File: translations/monero-core_it.ts (hunk 2 above)
Comment (erciccione) [reply translations/monero-core_it.ts]: For consisency with the source text

**Pull request merged**